### PR TITLE
Fix code after `.desktop` file renaming

### PR DESCRIPTION
**Pull request by AlexWayfer**
## Description 

`.desktop` file was renamed, but not its filename in the code.

### Motivation and Context

Resolve https://github.com/obsproject/obs-studio/pull/2100#issuecomment-542964293

### How Has This Been Tested?

Nohow.

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [ ] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [ ] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.

### CODE REVIEW

File: UI/xdg-data/com.obsproject.Studio.appdata.xml.in [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
   <id>com.obsproject.Studio</id>
```
Comment (AlexWayfer) [inline UI/xdg-data/com.obsproject.Studio.appdata.xml.in]: I'm not sure should I change this line.

**Comment by AlexWayfer** [comment]
Also, maybe we should have some tests for preventing such errors in compilation step.

**Comment by dionney** [comment]
This still gives me an error on install. The file is named "obs.desktop", not "com.obs.desktop".

File: UI/xdg-data/com.obsproject.Studio.appdata.xml.in (hunk 1 above)
Comment (eszlari) [reply UI/xdg-data/com.obsproject.Studio.appdata.xml.in]: No, please read the spec:

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html

**Comment by eszlari** [comment]
The original PR (https://github.com/obsproject/obs-studio/pull/2100) is questionable, because adding OBS to favorites works here under Ubuntu 19.10 (with GNOME 3.34) without renaming the .desktop file. I guess @AlexWayfer uses an older version of GNOME, where this has not been fixed.

**Comment by AlexWayfer** [comment]
> I guess @AlexWayfer uses an older version of GNOME, where this has not been fixed.

No, I'm using Arch Linux, GNOME is always the latest version. 3.34. for now.

File: UI/xdg-data/com.obsproject.Studio.appdata.xml.in (hunk 1 above)
Comment (AlexWayfer) [reply UI/xdg-data/com.obsproject.Studio.appdata.xml.in]: Thank you.

**Comment by AlexWayfer** [comment]
> This still gives me an error on install. The file is named "obs.desktop", not "com.obs.desktop".

Yeah, sorry, changed.

**Comment by eszlari** [comment]
@AlexWayfer Does it work for you, if you add `StartupWMClass=obs` to the .desktop file instead of renaming it?

**Comment by AlexWayfer** [comment]
> @AlexWayfer Does it work for you, if you add `StartupWMClass=obs` to the .desktop file instead of renaming it?

I'll try it later, in the evening on the days.

**Comment by AlexWayfer** [comment]
> @AlexWayfer Does it work for you, if you add StartupWMClass=obs to the .desktop file instead of renaming it?

Yes! But it shows in dock with delay about 10 seconds.

**Comment by AlexWayfer** [comment]
I'll create a new PR with reverting and suggested change. Thank you.

**Pull request closed without merging**